Frontend/src/app/landing/landing.component.ts

```diff
@@ -14,7 +14,7 @@ export class LandingComponent implements OnInit {
   codeType;
   showLoader: boolean = false;
 
-  constructor(private activatedRoute: ActivatedRoute, private helperService: HelperService, private sharedService: SharedService, public router: Router) {
+  constructor(private activatedRoute: ActivatedRoute, private sharedService: SharedService, public router: Router) {
     activatedRoute.params.subscribe((params) => {
```

Comment:
This will cause a subscription leak because its not being unsubscribed anywhere.

Reply:
Fixed: Added an unsubscribe() on ngOnDestory.